hawk/config/application.rb

```diff
@@ -46,5 +46,24 @@ class Application < Rails::Application
 
     config.x.hawk_is_sles = system("cat /etc/os-release | grep 'ID=.*sles' >/dev/null 2>&1")
 
+    def lookup_daemon_dir
+      [
+        "/usr/libexec/pacemaker",
+        "/usr/lib64/pacemaker",
+        "/usr/lib/pacemaker",
+        "/usr/lib64/heartbeat",
+        "/usr/lib/heartbeat"
+      ].each do |dir|
+        [
+          "crmd",
+          "pacemaker-controld",
```

Comment:
@diegoakechi  @petrpavlu @gao-yan I was wondering if `pacemaker-controld` is not backward compatible

Reply:
Although the names of the daemons have been changed from SLE15 SP1 as of:

https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Daemon_Changes

, we still maintain the old names as symbol links to the new executables.

Other than names, the supported options remain the same. 

AFAIK, what hawk and crmsh are concerned about is invoke these daemons' `metadata` command and retrieve the metadata. I'm pretty sure this still works all the same way.

Reply:
nice thx lot @gao-yan